app/src/main/java/assemble/controllers/UserController.java

```diff
@@ -0,0 +1,68 @@
+package assemble.controllers;
+
+import assemble.application.UserService;
+import assemble.domain.User;
+import assemble.dto.UserModificationData;
+import assemble.dto.UserRegistrationData;
+import assemble.dto.UserResultData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+/**
+ * 사용자 정보 컨트롤러.
+ */
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    /**
+     * 새로운 사용자 정보를 등록시킨다.
+     *
+     * @param registrationData 등록시킬 사용자 정보
+     * @return 등록된 사용자 정보
+     */
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    UserResultData create(@RequestBody @Valid UserRegistrationData registrationData) {
+        User user = userService.registerUser(registrationData);
+        return getUserResultData(user);
+    }
+
+    @PatchMapping("{id}")
+    UserResultData update(
+            @PathVariable Long id,
+            @RequestBody UserModificationData modificationData) {
+        return null;
+    }
+
+    @DeleteMapping("{id}")
+    void destroy(@PathVariable Long id) {
+        //
```

Comment:
이런 경우엔 TODO 태그를 남겨주시면 검색이 쉽습니다. IDE에서 관리해주기도 해요.
```suggestion
        // TODO ...
```